libs/ui-toolkit/src/components/toggle/toggle.tsx

```diff
@@ -0,0 +1,73 @@
+import classnames from 'classnames';
+import type { ChangeEvent } from 'react';
+
+// Supports controlled and uncontrolled setups.
+interface ToggleProps {
+  label: string;
+  value: string;
+}
+
+export type TogglesArr = [ToggleProps, ToggleProps, ...ToggleProps[]];
+
+export interface ToggleInputProps {
+  name: string;
+  toggles: TogglesArr;
```

Comment:
```suggestion
  toggles: ToggleProps[];
```

Reply:
We lose TS check for min 2 args doing this